tests/unit/grading/test_models.py

```diff
@@ -0,0 +1,33 @@
+from django.test import TestCase
+from django.utils import timezone
+
+from tests.unit.grading.mixins import DetectedPlagiarismData
+
+
+class GradingModelsTest(DetectedPlagiarismData, TestCase):
+    def setUp(self):
+        super().setUp()
+
+    def test_all_detected_plagiarisms_property(self):
+        self.assertTrue(
+            self.detected_plagiarism_alice
+            in self.plagiarism_test.all_detected_plagiarisms
+        )
+        self.assertTrue(
+            self.detected_plagiarism_bob
+            in self.plagiarism_test.all_detected_plagiarisms
+        )
+        self.assertEqual(self.plagiarism_test.all_detected_plagiarisms.count(), 2)
+
+    def test_time_description(self):
```

Comment:
This test has to be removed, since this functionality will be covered already by Django tests.